tests/testthat/test-guess_numerical_mark.R

```diff
@@ -26,4 +26,8 @@ test_that("correctly identifies the grouping mark", {
   comma_grouping_period_decimal <- withr::local_tempfile()
   writeLines('investor_name,portfolio_name,isin,market_value,currency\nx,y,z," 1,200.25 ",USD', comma_grouping_period_decimal)
   expect_identical(guess_grouping_mark(comma_grouping_period_decimal), ",")
-})
+
+  row_with_no_value <- withr::local_tempfile()  # resolves issue #14
+  writeLines('investor_name,portfolio_name,isin,market_value,currency\nx,y,z,100,USD\n,,,,', row_with_no_value)
+  expect_identical(guess_grouping_mark(row_with_no_value), ",")
```

Comment:
I think this works as expected, but I'm a little confused. This is a comma-delimited file, and the market value is just `100`. Is it guessing that the grouping mark is `,`, because the file is not semi-colon-delimited?

Reply:
because it's indeterminate... `,` is the default (even in you don't set a grouping mark explicitly, `readr::read_csv()` will use `,` as the default)

in this specific case, no grouping mark and no decimal mark is used, so it actually doesn't matter what you set them to

note that a strict CSV file that uses a `,` grouping mark should look like...
```
col1,col2,col3
x,y,"1,000.23"
```
that is why the default grouping mark can/should be `,` even in a strict, truly-comma-delimited CSV (any string/value should be quoted if it contains a `,`)

Reply:
Ok awesome. That is kinda sorta what I thought anyway, but just wanted to be certain